[windows/perfmon] - Add match_by_parent_instance to the documenation (#47304)
elastic/beats: metricbeat/module/windows/perfmon/_meta/docs.md

**`refresh_wildcard_counters`**
:   A boolean option to refresh the counter list at each fetch. By default, the counter list will be retrieved at the starting time, to refresh the list at each fetch, users will have to enable this setting.

**`match_by_parent_instance`**

> :   A boolean option that causes all instances of the same parent to have the same instance value. In the above example, this will cause metrics for `svchost`, `svchost#1`, etc. to have an `instance`

Reviewer: ```suggestion
:   A boolean option that causes all instances of the same parent to have the same instance value. In the previous example, this causes metrics for `svchost`, `svchost#1`, and so on to have an `instance`
```
